openapi-2.0/generated/src/IO.Swagger.Test/Api/AssetApiTests.cs

```diff
@@ -79,22 +80,42 @@ public void CreateAssetTest()
         {
             string customerKey = Guid.NewGuid().ToString();
             string name = $"Automation POC {Guid.NewGuid()}";
-            string description = "Automation POC Description";
-
-            decimal? id = 273724;
-            var helperResponse = instance.GetAssetById(id);
+            string description = "Automation POC Description";  
+            
             var assetType = new AssetType(196, "textblock", "Text Block");
-            var asset = new Asset(customerKey, null, null, assetType, null, null, null, name, description);
+            var asset = new Asset(null, customerKey, null, null, assetType, null, null, null, name, description);
 
             var response = instance.CreateAsset(asset);
 
             Assert.IsInstanceOf<Asset>(response);
         }
 
         [Test]
-        public void DeleteAssetTest()
+        public void DeleteAssetByIdTest()
```

Comment:
Should be split in 2 tests:

1. Create Asset, Delete it, Retrieve it - assert on the retrieval of the deleted Asset.
2. Multiple deletes of the same Asset.